src/backend/aspen/api/views/tests/test_create_phylo_runs.py

```diff
@@ -31,53 +30,11 @@ async def test_create_phylo_run(
     location = location_factory(
         "North America", "USA", "California", "Santa Barbara County"
     )
-    pathogen = random_pathogen_factory()
-    sample = sample_factory(group, user, location, pathogen=pathogen)
-    repo = random_default_repo_factory(split_client)
-    repo_data = aligned_repo_data_factory(pathogen, repo)
-    uploaded_pathogen_genome_factory(sample, sequence="ATGCAAAAAA")
-    async_session.add(group)
-    async_session.add(repo_data)
-    await async_session.commit()
-
-    auth_headers = {"user_id": user.auth0_user_id}
-    data = {
-        "name": "test phylorun",
-        "tree_type": "targeted",
-        "samples": [sample.public_identifier],
-    }
-    res = await http_client.post(
-        f"/v2/orgs/{group.id}/pathogens/{pathogen.slug}/phylo_runs/",
-        json=data,
-        headers=auth_headers,
-    )
-    assert res.status_code == 200
-    response = res.json()
-    assert response["template_args"] == {}
-    assert response["workflow_status"] == "STARTED"
-    assert response["group"]["name"] == group.name
-    assert response["user"]["name"] == user.name
-    assert response["user"]["id"] == user.id
-    assert response["pathogen"]["slug"] == pathogen.slug
-    assert "id" in response
-
-
-async def test_create_phylo_run_mpx(
```

Comment:
We didn't need specific/separate SC2/MPX tests since our test now generates random pathogens.